js/index.js

```diff
@@ -1 +1,45 @@
 // Your code goes here
+
+
+
+//example// document.getElementById("myBtn").addEventListener("click", displayDate);
+
+//1
+let coolNav = document.querySelectorAll(".nav-link")
+coolNav.forEach(element => {
+    element.addEventListener('mouseover', ()=>{ 
+        element.style.color = 'blue';
+
+        setTimeout(function() {
+            element.style.color = "";
+        }, 1000);
+    })
+});
+
+//2
+let imageFun = document.getElementsByClassName(".intro img")
+imageFun.addEventListener('click', ()=>{
+    element.style.transform = "rotate(180deg)";
+})
```

Comment:
I would have liked to see more event listeners to show that you're getting reps in in preparation for Friday's Sprint Challenge. Please reach out after 20 minutes if you hit a wall. You'll never learn if you don't ask for help when you get stuck, so please, reach out